### PR TITLE
NIFI-13507 removed nifi prometheus bundle, moved prometheus extension…

### PR DESCRIPTION
**Pull request by joewitt**
… utils to web api.  Removed StandardGangliaReporter

# Summary

[NIFI-13507](https://issues.apache.org/jira/browse/NIFI-13507)

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking

- [ ] [Apache NiFi Jira](https://issues.apache.org/jira/browse/NIFI) issue created

### Pull Request Tracking

- [ ] Pull Request title starts with Apache NiFi Jira issue number, such as `NIFI-00000`
- [ ] Pull Request commit message starts with Apache NiFi Jira issue number, as such `NIFI-00000`

### Pull Request Formatting

- [ ] Pull Request based on current revision of the `main` branch
- [ ] Pull Request refers to a feature branch with one commit containing changes

# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

- [ ] Build completed using `mvn clean install -P contrib-check`
  - [ ] JDK 21

### Licensing

- [ ] New dependencies are compatible with the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0) according to the [License Policy](https://www.apache.org/legal/resolved.html)
- [ ] New dependencies are documented in applicable `LICENSE` and `NOTICE` files

### Documentation

- [ ] Documentation formatting appears as expected in rendered files

### CODE REVIEW

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/prometheusutil/AbstractMetricsRegistry.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.nifi.prometheus.util;
+package org.apache.nifi.prometheusutil;
```
Comment (exceptionfactory) [inline nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/prometheusutil/AbstractMetricsRegistry.java:17]: This change seems untentional, should it be reverted?

**Review by exceptionfactory — CHANGES_REQUESTED** [review]
Thanks @joewitt! The removals make sense, but there are a number of package changes from `prometheus.util` to `prometheusutil` that should be reverted.

The Maven configuration for `nifi-standard-reporting-tasks` also has a dependency on `metrics-ganglia` that should be removed. The `com.yammer.metrics:metrics-core` dependency also does not appear to be used outside of the Ganglia Reporter, so that can be removed.

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/prometheusutil/AbstractMetricsRegistry.java (hunk 1 above)
Comment (joewitt) [reply nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/prometheusutil/AbstractMetricsRegistry.java:17]: Yeah the new package name seems fine to me.  I didnt see value in it being further lower.

The stuff used to be in extension-utils and is no longer needed plus i'd like to see nothing in the framework folders know anything about extension paths so this was a win.

**Comment by joewitt** [comment]
should be addressed

**Review by exceptionfactory — APPROVED** [review]
Thanks for the additional dependency removals and package name clarification @joewitt, +1 merging

**Pull request closed without merging**